keras/engine/training.py

```diff
@@ -928,6 +929,23 @@ def run_eagerly(self):
     def run_eagerly(self, value):
         self._run_eagerly = value
 
+    @property
+    def distribute_reduction_method(self):
+        """Settable attribute indicating how the model should reduce
+        loss and metric values from replicas.
+
+        Default: `"auto"`. This should be good for general use cases.
+        It boils down to using `"sum"` or `"first"` conditioned on the
+        specific implementation of the `tf.distribute` strategy.
+        In case of a `tf.distribute.MirroredStrategy` it boils down to `"sum"`
```

Comment:
It looks to me that we don't have specific implementation for "In case of a `tf.distribute.MirroredStrategy` it boils down to `"sum"` to account for the case of custom training loops.", so I would omit this to avoid confusions.

Otherwise looks good. Thank you!

Reply:
yes. it is actually that is `"sum"` in all cases, but when TPU is used, when it is `"first"`. I will update the doc accordingly